test/integration/messaging/process-publish-message.test.js

```diff
@@ -78,10 +89,10 @@ describe('publish statement', () => {
     expect(mockSendEmail.mock.calls[0][1]).toBe(mockRequest.email)
   })
 
-  test('should send email with file link', async () => {
-    await processPublishMessage(message, receiver)
-    expect(mockSendEmail.mock.calls[0][2].personalisation.link_to_file).toBe(MOCK_PREPARED_FILE)
-  })
+  // test('should send email with file link', async () => {
+  //   await processPublishMessage(message, receiver)
+  //   expect(mockSendEmail.mock.calls[0][2].personalisation.link_to_file).toBe(MOCK_PREPARED_FILE)
+  // })
 
```

Comment:
Remove if not needed.

Reply:
Good spot -- removed to reduce the output from logs when recording the Notify response object.

Added back in and all passing